src/main/java/org/fundacionjala/pivotal/objects/EditProject.java

```diff
@@ -114,18 +114,18 @@ private void setTxtTask(final String check) {
      * @param values Map<DescriptionValues, Object>.
      * @return strategyMap.
      */
-    public Map<DescriptionValues, Meth> getStrategyStepMap(final Map<DescriptionValues, Object> values) {
-        final Map<DescriptionValues, Meth> strategyMap = new HashMap<>();
+    public EnumMap<DescriptionValues, Steps> getStrategyStepMap(final Map<DescriptionValues, String> values) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "Map" rather than the implementation "EnumMap". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)